INFO     root:test_calculator.py:132 Starting test for valid subtraction operation with login.
INFO     root:test_calculator.py:139 Attempting to log in with credentials: {'userName': 'admin', 'password': '123'}
INFO     root:test_calculator.py:141 Login response status code: 200
INFO     root:test_calculator.py:144 Login response data: {'message': None, 'data': {'success': True, 'token': '15adef6c-1dcc-4b66-815b-fa7782eecfb0'}, 'status': 'success'}
INFO     root:test_calculator.py:146 Login response data: {'message': None, 'data': {'success': True, 'token': '15adef6c-1dcc-4b66-815b-fa7782eecfb0'}, 'status': 'success'}
INFO     root:test_calculator.py:148 Received token: 15adef6c-1dcc-4b66-815b-fa7782eecfb0
INFO     root:test_calculator.py:156 Sending calculation request: {'op1': -5, 'operation': '-', 'op2': -8}
INFO     root:test_calculator.py:161 Received response with status code: 200
INFO     root:test_calculator.py:164 Calculation response data: {'message': None, 'data': 3, 'status': 'success'}
INFO     root:test_calculator.py:170 The result of the addition operation is correct.